index.html

```diff
@@ -14,6 +14,138 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  
+  <div class="container">
+      <header>
+    <nav>
+      <a href="">Resume</a>
+      <a href="">Portfolio</a>
+      <a href="">Contact Me</a>
+      <a href="">Links</a>
+
+    </nav>
+  <h2>Thomas Lafyette Utsey <br>
+      182 24th st NW <br>
+      Hickory NC 28601 <br>
+      (843)701-2055 <br>
+      tom@edgegluer.com </h2>
+    </header>
+    <section class="intro">
+
+      <img src="Capture.png" alt="A picture of Thomas Utsey" class="myimg">
```

Comment:
Include a real image next time, even if it's just a placeholder picture. It is a good habit to get in!